source/wp-content/themes/wporg-developer-2023/src/style/style.scss

```
@@ -14,3 +14,68 @@
 .wporg-filtered-search-form {
 	display: block;
 }
+
+.wp-block-search.wp-block-search__button-inside .wp-block-search__inside-wrapper {
+	--wp--custom--button--hover--color--text: var(--wp--preset--color--charcoal-1);
+	--wp--custom--button--hover--color--background: var(--wp--preset--color--light-grey-2);
+	padding: 0;
+	border: none;
+	border-radius: var(--wp--custom--button--border--radius);
+	background: var(--wp--preset--color--white);
+	color: var(--wp--preset--color--charcoal-1);
+
+	.wp-block-search__input {
+		padding-right: 0;
+		border: none;
+	}
+
+	.wp-block-search__button {
+		padding-top: calc(var(--wp--preset--spacing--20) * 0.5);
+		padding-right: calc(var(--wp--preset--spacing--20) * 0.75);
+		padding-bottom: calc(var(--wp--preset--spacing--20) * 0.5);
+		padding-left: calc(var(--wp--preset--spacing--20) * 0.75);
+
+		background: transparent;
+
+		svg {
+			fill: inherit;
+		}
+	}
+}
+
+@media (max-width: 781px) {
+	.wp-block-column[style*="border-right-"] {
+		border-bottom: 1px solid var(--wp--preset--color--light-grey-1);
+		border-right: none !important;
+	}
+
+	.wp-block-column[style*="border-right-"][style*="padding-right"] {
+		margin-left: calc(var(--wp--preset--spacing--edge-space) *- 1) !important;
+		margin-right: calc(var(--wp--preset--spacing--edge-space) *- 1) !important;
```

My Sass build blows up with this, changing it to this fixes it:
```suggestion
		margin-left: calc(var(--wp--preset--spacing--edge-space) * -1) !important;
		margin-right: calc(var(--wp--preset--spacing--edge-space) * -1) !important;
```

---

Very weird: I copied that block directly from the [documentation style](https://github.com/WordPress/wporg-documentation-2022/blob/13f348f8949c355c861b8e9354cf4a2d65dedb78/source/wp-content/themes/wporg-documentation-2022/src/style/style.scss#L128) and made no updates. A linter must have reordered it... scary.